src/Components/Comments.jsx

```diff
@@ -22,6 +28,18 @@ useEffect(() => {
   })
   }, []);
 
+  const handleClick = e => {
+      setshowPostButton(false);
+    api.postNewComment(articleId, newComment).then(comment => {
+      setShowTextArea(false);
+      setNewComment('');
+      setShowSuccessMsg(true);
+      setTimeout(() => setShowSuccessMsg(false), 6000);
```

Comment:
Doesnt seem to be any error handling if it doesnt post. At the moment, the button just disappears, but nothing else